ephios/core/templates/core/event_list.html

```diff
@@ -72,38 +88,197 @@ <h1>{% translate "Events" %}
+                    <button class="btn btn-primary btn-sm dropdown-toggle d-none d-md-inline-block" type="button"
+                            id="eventCreateButton"
+                            data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                        <span class="fa fa-plus"></span>
+                        {% translate "Add event" %}
+                    </button>
+                    <div class="dropdown-menu dropdown-menu-right" aria-labelledby="eventCreateButton">
+                        {% for eventtype in eventtypes %}
+                            <a class="dropdown-item" href="{% url "core:event_create" eventtype.pk %}">
+                                {% blocktranslate trimmed with title=eventtype.title %}
+                                    Add {{ title }}
+                                {% endblocktranslate %}</a>
+                        {% endfor %}
                     </div>
-                {% endif %}
-                <a class="btn btn-secondary ms-auto"
-                   href="?{% param_replace date=next_month_first|date:"Y-m-d" %}"
-                >{{ next_month_first|date:"F Y" }} <i class="fas fa-arrow-right"></i></a>
+                </div>
+            {% endif %}
+
+            <div class="d-flex justify-content-between mt-2">
+                <a class="btn btn-secondary"
+                   href="?{% param_replace date=previous_date|date:"Y-m-d" %}"
+                ><i class="fas fa-arrow-left"></i>
+                    {% if mode == "calendar" %}
+                        {{ previous_date|date:"F Y" }}
+                    {% elif mode == "day" %}
+                        {{ previous_date|date:"d. M" }}
+                    {% endif %}
+                </a>
+
+                <h3>
+                    {% if mode == "calendar" %}
+                        {{ date|date:"F Y" }}
+                    {% elif mode == "day" %}
+                        {{ date|date:"l, j. F Y" }}
+                    {% endif %}
+                </h3>
+
+                <a class="btn btn-secondary"
+                   href="?{% param_replace date=next_date|date:"Y-m-d" %}"
+                >
+                    {% if mode == "calendar" %}
+                        {{ next_date|date:"F Y" }}
+                    {% elif mode == "day" %}
+                        {{ next_date|date:"d. M" }}
+                    {% endif %}
+                    <i class="fas fa-arrow-right"></i></a>
             </div>
         {% endif %}
+
+
     </div>
 
     {% if mode == "calendar" %}
         {{ calendar }}
+    {% elif not event_list %}
+        <div class="mb-3">
+            {% if mode == "day" %}
+                <h3>{{ date|date:"l, j. F Y" }}</h3>
+            {% endif %}
+            <h5 class="mt-5 text-center">
+                {% translate "No results." %}
+            </h5>
+        </div>
+    {% elif mode == "day" %}
+        <style nonce="{{ request.csp_nonce }}">
```

Comment:
can you move the list and day-grid both into their own template fragment to make this file smaller?

Reply:
Done